dashboard/test/models/user_test.rb

```diff
@@ -734,6 +734,23 @@ class UserTest < ActiveSupport::TestCase
     assert_equal '21+', user.age
   end
 
+  test 'changing from student to teacher clears terms_of_service_version' do
+    user = create :student, terms_of_service_version: 1
+    user.update!(user_type: User::TYPE_TEACHER, email: 'tos@example.com')
+    assert_nil user.terms_of_service_version
+  end
+
+  test 'changing from teacher to student does not clear terms_of_service_version' do
```

Comment:
@poorvasingal are we okay not clearing the TOS flag when switching from teacher to student? Does the teacher TOS cover everything that the student one would?